docs/blog/posts/using-keptn-analyses-in-argo-workflows.md

```diff
@@ -0,0 +1,174 @@
+---
+date: 2024-05-04
+authors: [bacherfl]
+description: >
+  In this blog post you will learn how to use the Analysis feature of Keptn within Argo Workflows.
+categories:
+  - SRE
+  - Application Performance
+  - Analysis
+comments: true
+---
+
+# Using Keptn Analyses in Argo Workflows
+
+In this blog post we will explore how Keptn
+can be used within [Argo Workflows](https://argoproj.github.io/workflows/) to
+execute load tests using a [KeptnTask](../../docs/reference/crd-reference/task.md), and analyze their results
+afterward using the [Keptn Analysis](../../docs/reference/crd-reference/analysis.md) feature.
+Argo Workflows is a workflow orchestration engine built for Kubernetes, which makes it a perfect fit
+to be used together with Keptn.
+We will achieve this by using the support for [Executor Plugins](https://argo-workflows.readthedocs.io/en/latest/executor_plugins/)
+provided by Argo Workflows.
+This plugin mechanism is an easy way of integrating custom functionality into an Argo Workflow,
+and providing further nodes in the workflow with valuable information, such as the outcome of a Keptn Analysis.
+
+<!-- more -->
+
+## Technologies used for this example
+
+For this, we are using the following technologies:
+
+- The [Keptn Analysis](../../docs/reference/crd-reference/analysis.md) resource
+  defines goals for our metrics and evaluates them.
+- [Argo Workflows](https://argoproj.github.io/workflows/) is the workflow engine.
+- [Prometheus](https://prometheus.io): Provides monitoring data for the application.
+
+## Architecture
+
+The overall architecture of this example is depicted in the diagram below:
+
+![Architecture](./using-keptn-analyses-in-argo-workflows/workflow.png)
+
+Our example workflow exists of two nodes, which do the following:
+
+- The **Load Tests** node creates a `KeptnTask` that executes load tests against
```

Comment:
Style question for the team: Should this be "...creates a `KeptnTask` resource that executes..."?  I think @odubajDT and @mowies  had requested this in other docs.